core-blocks/button/index.js

```diff
@@ -76,17 +72,16 @@ export const settings = {
 
 	attributes: blockAttributes,
 
-	getEditWrapperProps( attributes ) {
-		const { align } = attributes;
+	getEditWrapperProps() {
 		const props = { 'data-resized': true };
```

Comment:
I double checked it, it looks like `data-resized` is no longer used for the button. Can you confirm and remove it if that is true?

Reply:
It does exist in `master`, but removing it doesn't seem to be affecting anything. Neither does it seem to be taking effect anywhere in our branch. I've removed it in [this commit](https://github.com/WordPress/gutenberg/pull/7100/commits/1570301e8c0e0126fc2749b89cea801b47f1d844).